dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -7153,6 +7151,22 @@ public List<Contentlet> findAllVersions(Identifier identifier, boolean bringOldV
         return contentlets;
     }
 
+    @CloseDBIfOpened
+    @Override
+    public List<Contentlet> findAllVersions(final Identifier identifier, final boolean bringOldVersions, final User user, final boolean respectFrontendRoles, final int limit, final int offset)
```

Comment:
some this is a new method, a buider pattern for the parameter would be nice

Reply:
Done!